x-pack/plugins/infra/common/http_api/snapshot_api.ts

```diff
@@ -18,13 +18,7 @@ export const SnapshotNodePathRT = rt.intersection([
   rt.partial({
     ip: rt.union([rt.string, rt.null]),
     os: rt.union([rt.string, rt.null]),
-    cloudProvider: rt.union([
-      rt.literal('gcp'),
-      rt.literal('aws'),
-      rt.literal('azure'),
-      rt.literal('unknownProvider'),
-      rt.null,
-    ]),
+    cloudProvider: rt.union([rt.string, rt.null]),
```

Comment:
@jennypavlova you were right here. it would be hard to maintain new/unexpected values. Luckily we caught the problem using Slingshot

Reply:
Thanks for finding that! Yes, it's usually tricky when you have certain fields and usually, the value can be something different and not the expected one (also we have a fallback icon so at least it won't be empty)